D1673/P1673.md

```diff
@@ -6529,40 +6557,61 @@ private:
 
   * [2.3]{.pnum} `Accessor` meets the accessor policy requirements **[mdspan.accessor.reqmts]**.
 
+  * [2.4]{.pnum} _`scalable`_`<ScalingFactor, element_type>` is `true`. 
+
+  * [2.5]{.pnum} `is_constructible_v<element_type, decltype(declval<ScalingFactor>() * element_type)>` is `true`.
```

Comment:
I'm not sure if we need this.  The definition of `scaled_accessor`'s `element_type` is "the result of the product of the scaling factor and the nested accessor's `element_type`."  If the product expression isn't well formed, then the definition of `element_type` isn't well-formed either, so we can't even talk about `element_type` as a requirement of `scaled_accessor`.

Reply:
Oh this needed to be `using element_type = add_const_t<Accessor::element_type>;`